Objective-C/CBLReplicator.mm

```diff
@@ -454,6 +455,36 @@ - (void) removeChangeListenerWithToken: (id<CBLListenerToken>)token {
     }
 }
 
+- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error {
+    if (_config.replicatorType > 1) {
+        if (error)
```

Comment:
Is this right?  If there is already an error, replace it?

Reply:
this is to make sure, the passed in argument to this function is valid, then only we will assign value to the pointed location.

Reply:
All good, then!